NitroxClient/GameLogic/MobileVehicleBay.cs

```diff
@@ -20,20 +26,27 @@ public MobileVehicleBay(IPacketSender packetSender, Vehicles vehicles)
 
         public void BeginCrafting(ConstructorInput constructor, GameObject constructedObject, TechType techType, float duration)
         {
-            NitroxId constructorId = NitroxEntity.GetId(constructor.gameObject);
-
-            Log.Debug($"Building item from constructor with id: {constructorId}");
+            if (!TransmitLocalSpawns)
+            {
+                return;
+            }
 
-            VehicleModel vehicleModel = vehicles.BuildVehicleModelFrom(constructedObject, techType);
-            vehicles.AddVehicle(vehicleModel);
+            // Sometimes build templates, such as the cyclops, are already tagged with IDs.  Remove any that exist to retag.
+            UnityEngine.Component.DestroyImmediate(constructedObject.GetComponent<NitroxEntity>());
```

Comment:
Think we should prefix this with "TODO" here. (Figure out why NitroxEntity is in the template). Recreating the NitroxEntity can be problematic if NitroxEntity gets more importance/state in the future.